Block/StaticBlock.php

```diff
@@ -80,18 +74,27 @@ private function createPrismicDocument()
     /**
      * @return string
      * @throws ContextNotFoundException
```

Comment:
This is incorrect, I think since this Exception class is part of your catch now.

Reply:
Changed it back, removed the try and catch under the first one because that will already throw the documentnotfoundexception